sportsdataverse/cfb/cfb_pbp.py

```
@@ -1597,6 +1622,133 @@ def __helper_cfb_pbp_features(self, pbp_txt, init):
+                .then(pl.when(pl.col("season") > 2013).then(pl.lit(75)).otherwise(pl.lit(80)))
+                .when(
+                    (pl.col("type.text").is_in(kickoff_vec).or_(pl.col("text").str.contains(r"(?i)kickoff")))
+                    .and_(pl.col("text").str.contains(r"(?i)penalty"))
+                    .and_(
+                        pl.col("text").str.contains(
+                            r"(?i)for \d+ yds|touchback|return|out of bounds|recovered|downed|fair catch",
+                        )
+                        == False
+                    )
+                    .and_(pl.col("start.pos_team.id").shift(-1) == pl.col("end.pos_team.id"))
+                    .and_(pl.col("start.yardsToEndzone").shift(-1).is_null() == False),
```

Flake8 (7.3.0)

[error] comparison to False should be 'if cond is False:' or 'if not cond:'

(E712)